docs/concepts-broker-load-balancing-migration.md

```diff
@@ -0,0 +1,170 @@
+
+:::
+
+### Change broker.conf file 
+
+1. [Upgrade the Pulsar cluster](./helm-upgrade.md) to 3.0.x.
+
+2. Access to the broker.conf file.
+
+    ```bash
+    vim apache-pulsar-@pulsar:version@/conf/broker.conf
+    ```
+
+3. Change the following settings in the broker.conf file:
+
+   - Update [broker load balancer](./concepts-broker-load-balancing-overview.md) by setting [loadManagerClassName](https://github.com/apache/pulsar/blob/69d7a2bf14555f11a716a9545c5cf391d8179a27/conf/broker.conf#L1309C20-L1309C20) to `ExtensibleLoadManagerImpl`. 
+
+   - Update [bundle unloading strategy](./concepts-broker-load-balancing-concepts.md#bundle-unloading-strategies) by setting `loadBalancerLoadSheddingStrategy` to `TransferShedder`. 
+
+    ```conf
+    loadManagerClassName=org.apache.pulsar.broker.loadbalance.extensions.ExtensibleLoadManagerImpl 
+
+    loadBalancerLoadSheddingStrategy=org.apache.pulsar.broker.loadbalance.extensions.scheduler.TransferShedder
+    ```
+
+    :::note
+
+      - The [extensible broker load balancer](./concepts-broker-load-balancing-types.md) is **available in Pulsar 3.0.0** or later.
+      - The [TransferShedder unloading strategy](./concepts-broker-load-balancing-concepts.md#bundle-unloading-strategies) is **only available** in the [extensible load balancer](./concepts-broker-load-balancing-types.md).
+  
+    :::
+
+4. Restart the Pulsar cluster. The new settings will take effect after the restart.  
+
+## Migrate from extensible to modular broker load balancer
+
+You can migrate from the extensible to the modular broker load balancer, by manually changing the setting in the broker.conf file.
+
+:::note
+
+The pulsar-admin tool is not supported for this migration.
+
+:::
+
+### Change broker.conf file 
+
+1. Access to the broker.conf file.
+
+    ```bash
+    vim apache-pulsar-@pulsar:version@/conf/broker.conf
+    ```
+
+2. Change the following settings in the broker.conf file:
+   
+   - Update broker load balancer by setting [loadManagerClassName](https://github.com/apache/pulsar/blob/69d7a2bf14555f11a716a9545c5cf391d8179a27/conf/broker.conf#L1309C20-L1309C20) to ModularLoadManagerImpl
+    
+   - Update [bundle unloading strategy](./concepts-broker-load-balancing-concepts.md#bundle-unloading-strategies) to OverloadShedder, ThresholdShedder, or UniformLoadShedder based on your needs.
+
+    ```
+    loadManagerClassName=org.apache.pulsar.broker.loadbalance.impl.ModularLoadManagerImpl
+
+    loadBalancerLoadSheddingStrategy=org.apache.pulsar.broker.loadbalance.impl.ThresholdShedder
+    ```
+
+    :::note
+
+    [TransferShedder](./concepts-broker-load-balancing-concepts.md#bundle-unloading-strategies) is **only supported** in the extensible broker load balancer, so you need to change TransferShedder to other [bundle unloading strategies](./concepts-broker-load-balancing-concepts.md#bundle-unloading-strategies).
+
+    :::
+
+3. Restart the Pulsar cluster. The new settings will take effect after the restart.  
+
```

Comment:
@Demogorgon314 

I think we need to explain how Pulsar optimizes the lookup/assignment during this migration. `Topic Lookup and Assignment Redirection during Migration` Please provide the migration logic as you worked on this part.

For example,
During migration from modular to extensible, the lookup and assignment will be redirected to the brokers with the extensible load balancer.

During migration from extensible to modular, the lookup and assignment will be redirected to the brokers with the modular load balancer.

Reply:
OK, updated in https://github.com/apache/pulsar-site/pull/657/files#diff-2abdb541a434c7250793845da41718b322344e45cc83098c3c57addacc4b5a6aR81 and https://github.com/apache/pulsar-site/pull/657/files#diff-2abdb541a434c7250793845da41718b322344e45cc83098c3c57addacc4b5a6aR122